scripts/generate_api_docs.sh

```diff
@@ -6,5 +6,5 @@ cd ${script_dir}/..
 
 pip install -U pdoc3
 rm -rf docs/api-docs
```

Comment:
I believe this script needs to remove the old generated docs

```suggestion
rm -rf docs/static/api-docs
```